WorldSvr/OnWarehouse.c

```diff
@@ -65,10 +97,6 @@ CLIENT_PROCEDURE_BINDING(WAREHOUSE_CURRENCY_DEPOSIT) {
 
 	SocketSend(Socket, Connection, Response);
 	return;
-
-error:
-	memcpy(&Character->Data.WarehouseInfo, &kWarehouseInfoBackup, sizeof(struct _RTCharacterWarehouseInfo));
```

Comment:
Delete the one on the func entry also its useless
`memcpy(&kWarehouseInfoBackup, &Character->Data.WarehouseInfo, sizeof(struct _RTCharacterWarehouseInfo));`